test/PowerShellEditorServices.Test/Language/SymbolsServiceTests.cs

```diff
@@ -952,5 +959,30 @@ public void FindsSymbolsInNoSymbolsFile()
             IEnumerable<SymbolReference> symbolsResult = FindSymbolsInFile(FindSymbolsInNoSymbolsFile.SourceDetails);
             Assert.Empty(symbolsResult);
         }
+
+        [Fact]
+        public void FindsRegionSymbolsInFile()
```

Comment:
This technically doesn't belong here as SymbolsService isn't used for DocumentSymbolHandler. Move to own test-file? Suggestion?